templates/home.php

```diff
@@ -19,8 +19,19 @@
 	),
 );
 ?>
-
 <div class="event-page-wrapper">
+	<div class="notice" style="padding: .5rem">
+		<?php
+		echo wp_kses(
+			sprintf(
+			// translators: %s is a link to a page about hosting events.
+				__( 'Do you want to host your own event? <a href="%s">Find more information here</a>.', 'gp-translation-events' ),
+				'https://make.wordpress.org/polyglots/2024/05/29/translation-events-inviting-gtes-to-create-and-manage-events/'
```

Comment:
The post only has information on how GTEs can create events, right? If the user reading this message is not a GTE, I think they will be confused because they will follow the link but will not find an answer on how they can create an event.

Reply:
The link is somewhat temporary (also inserted via a variable) as long we don't have a Polyglots Handbook page. I agree it's not great but this is the best that we have right now.